src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -137,7 +139,7 @@ public EditModuleDescriptor(EditModuleDescriptor toCopy) {
          * Returns true if at least one field is edited.
          */
         public boolean isAnyFieldEdited() {
-            return CollectionUtil.isAnyNonNull(code, credits, tags);
+            return CollectionUtil.isAnyNonNull(code, title, credits, tags);
```

Comment:
Minor nit: EditModuleDescriptorTest does not currently cover this portion of the code under test coverage, can consider testing this portion out, but not that important HAHAHA.